Makefile

```diff
@@ -226,67 +223,35 @@ build-all-in-one-linux: build-ui
 
```

Comment:
is `GOARCH=amd64 ` missing above?

Reply:
Not really, this target should have an open `GOARCH`